app/controllers/art_labels_controller.rb

```diff
@@ -32,4 +31,10 @@ def create
   def art_label_params
     params.require(:art_label).permit(:name, :image_url, :brewery, :beer_style, :art_style, :container_type, :beer_description, :art_description, :beer_rating)
   end
+
+  def authorize_user
+    if !user_signed_in? || !current_user.admin?
+      raise ActionController::RoutingError.new("Not Found (Admin)")
+    end
+  end
```

Comment:
You may want to move this to a place where it can easily be reused. See my gist posted yesterday about where to store these helper methods (hint: in the ApplicationController)